src/udev/net/ethtool-util.h

```diff
@@ -51,6 +53,18 @@ typedef enum NetDevFeature {
         _NET_DEV_FEAT_INVALID = -1
 } NetDevFeature;
 
+typedef enum NetDevPort {
+        NET_DEV_PORT_TP     = 0x00,
+        NET_DEV_PORT_AUI    = 0x01,
+        NET_DEV_PORT_MII    = 0x02,
+        NET_DEV_PORT_FIBRE  = 0x03,
+        NET_DEV_PORT_BNC    = 0x04,
+        NET_DEV_PORT_DA     = 0x05,
+        NET_DEV_PORT_NONE   = 0xef,
+        NET_DEV_PORT_OTHER  = 0xff,
```

Comment:
Sorry to ask that, but what NET_DEV_PORT_NONE and NET_DEV_PORT_OTHER are there for ? They look unused, this will increase the size of port_table for nothing